src/Entity/ShopBillingData.php

```diff
@@ -37,19 +37,73 @@ class ShopBillingData implements ShopBillingDataInterface
     /** @var string|null */
     protected $postcode;
 
-    public function __construct(
-        ?string $company,
-        ?string $taxId,
-        ?string $countryCode,
-        ?string $street,
-        ?string $city,
-        ?string $postcode
-    ) {
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function setId(?int $id): void
```

Comment:
Similar to `CustomerBillingData`, remove the type of id not to force integers
```suggestion
    public function getId()
    {
        return $this->id;
    }

    public function setId($id): void
```